examples/03-Circuit/EMIT_HFSS_Example.py

```diff
@@ -0,0 +1,101 @@
+"""
+EMIT Example
+------------
+This tutorial shows how you can use PyAEDT to open an AEDT project with
+an HFSS design, create an EMIT design in the project, then link the HFSS design
+as a coupling link in the EMIT design.
+"""
+# sphinx_gallery_thumbnail_path = 'Resources/emit.png'
+import os
+import tempfile
+
+# Import required modules
+from pyaedt.generic.filesystem import Scratch
+
+# Setup paths for module imports
+from _unittest.conftest import local_path, scratch_path
+
+from pyaedt import Emit
+from pyaedt import Desktop
+
+###############################################################################
+# Initialization Settings
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Change NonGraphical Boolean to False to open AEDT in graphical mode
+# With NewThread = False, an existing instance of AEDT will be used, if
+# available. This example will use AEDT 2022.2
+NonGraphical = False
+NewThread = False
+desktop_version = "2022.2"
+
+
+###############################################################################
+# Launch AEDT and EMIT Design
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Desktop class initializes AEDT and starts it on specified version and
+# specified graphical mode. NewThread Boolean variable defines if a user wants
+# to create a new instance of AEDT or try to connect to existing instance of
+# it.
+d = Desktop(desktop_version, NonGraphical, NewThread)
+
+tmpfold = tempfile.gettempdir()
+
+temp_folder = os.path.join(tmpfold, ("EmitHFSSExample"))
+if not os.path.exists(temp_folder):
+    os.mkdir(temp_folder)
+
+example_name = "Cell Phone RFI Desense"
+example_aedt = example_name + ".aedt"
+example_lock = example_aedt + ".lock"
+
+example_project = os.path.join(local_path, "example_models", example_aedt)
```

Comment:
@jianwei425 the examples files of published examples have not to be in that folder. On monday I can explain where to put it and to download from Examples.

Reply:
decided to copy it from the EDT install and not saving a copy of the example file in pyaedt